src/main/java/io/appium/java_client/pagefactory/README.md

```diff
@@ -0,0 +1,69 @@
+# Standard Selectors
+
+## AndroidFindBy / iOSXCUITFindBy / WindowsFindBy
+
+# Advanced Selectors
+
+## iOS's Class Chain Queries
+
+Our XCUiTest integration has full support for the 'Class Chain' concept. This
+can do much of what XPath does...but faster.
+
+### External References
+
+Refer to [the official WebDriverAgent query docs](https://github.com/facebookarchive/WebDriverAgent/wiki/Class-Chain-Queries-Construction-Rules)
+to learn more about the general concept.
+
+### Sample usage:
+
```

Comment:
please use markdown formatters for code blocks: https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks

Reply:
Certainly. I'll get right on that.